web/src/lib/components/admin-page/jobs/job-tile.svelte

```diff
@@ -94,7 +96,15 @@
     </div>
   </div>
   <div class="flex w-full flex-row overflow-hidden sm:w-32 sm:flex-col">
-    {#if !isIdle}
+    {#if disabled}
+      <JobTileButton
+        disabled={true}
+        color="light-gray"
+        on:click={() => dispatch('command', { command: JobCommand.Start, force: false })}
+      >
+        <AlertCircle size="48" /> DISABLED
```

Comment:
Can we make the size a bit smaller? 36 is probably good

Reply:
Sure, I just copied it from the play icon.